feature/gribi/ate_tests/gribi_scaling/gribi_scaling_test.go

```diff
@@ -289,87 +289,66 @@ func pushDefaultEntries(t *testing.T, args *testArgs, nextHops []string) []strin
 	return virtualVIPs
 }
 
-// createVrf creates takes in a list of VRF names and creates them on the target devices.
-func createVrf(t *testing.T, dut *ondatra.DUTDevice, d *oc.Root, vrfs []string) {
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
+	dp1 := dut.Port(t, "port1")
+	dp2 := dut.Port(t, "port2")
+	d := &oc.Root{}
+
+	vrfs := []string{*deviations.DefaultNetworkInstance, vrf1, vrf2, vrf3}
+	createVrf(t, dut, vrfs)
+
+	// configure Ethernet interfaces first
+	configureInterfaceDUT(t, d, dut, dp1, "src")
+	configureInterfaceDUT(t, d, dut, dp2, "dst")
+
+	// configure an L3 subinterface without vlan tagging under DUT port#1 and assign it to vrf1
+	createSubifDUT(t, d, dut, dp1, 0, 0)
+	fptest.AssignToNetworkInstance(t, dut, dp1.Name(), vrf1, 0)
```

Comment:
This pr introduce a breakage for us. can u please add a deviation in AssignToNetworkInstance to use the interface id exactly as interface name for subinterface zero. 

```
	if si==0 & *deviations.SubInterfaceIDZero{
		netInstIntf.Id = ygot.String(intf.GetName()))
	} else {
		  netInstIntf.Id = ygot.String(intf.GetName() + "." + fmt.Sprint(si))
	}

Reply:
Ok. That would be the new deviation right?

Thanks for the feedback @mojiiba

Reply:
Thanks @mojiiba. I don't see the deviation existing. Do you plan to create one? 

Or, do you prefer us to follow the same way as you did in https://github.com/openconfig/featureprofiles/pull/1181/files#diff-22fbd408bbd0345dd7d80fd32959110220f47eda63668a9891b92c31d46ba23bR212 ?

Reply:
Added a `deviation_subif_id_zero` as suggested above, please take a look

Reply:
thanks a lot!